.travis.yml

```diff
@@ -9,6 +9,7 @@ before_install:
   - nvm install 7.10.0
   - npm install
   - npm run build:js
+  - awk '!/jekyll-admin/' Gemfile > temp && mv temp Gemfile
```

Comment:
Yeah, I like adding the comment in there and I changed the command to `awk '!/jekyll-admin/' Gemfile > Gemfile.temp && mv Gemfile.temp Gemfile` to hopefully be a little more readable.

`awk '!/jekyll-admin/' Gemfile > Gemfile` unfortunately doesn't work as it overwrites the file and generates a blank `Gemfile`...